voorhoede/tudelft-spacefinder
# Feat/custom spacefinder mapbox style

# Changes


- Add custom mapbox style

# Associated issue



# How to test


1. Open preview link
2. Click to the map
3. Check if style is loaded

# Checklist
